src/main/java/sparksoniq/semantics/visitor/RuntimeIteratorVisitor.java

```diff
@@ -192,7 +191,7 @@ public RuntimeIterator visitProlog(Prolog expression, RuntimeIterator argument)
     public RuntimeIterator visitFlowrExpression(FlworExpression expression, RuntimeIterator argument) {
         FlworClause startClause = expression.getStartClause();
         RuntimeTupleIterator previous = null;
-        previous = this.visitFlowrClause(startClause, argument, previous);
+        previous = this.visitFlowrClause(startClause, argument, null);
```

Comment:
This, unlike the rest, looks like a material change. Could you please explain it to me in a few words? Thanks!

Reply:
addressed offline, it's all good